operations/service_deploy_smoke.go

```diff
@@ -42,7 +42,7 @@ func startLocalEvergreen() cli.Command {
 				return errors.Wrap(err, "couldn't get working directory")
 			}
 			binary := filepath.Join(wd, "clients", runtime.GOOS+"_"+runtime.GOARCH, "evergreen")
-			if err := smokeRunBinary(exit, "web.service", wd, binary, "service", "web", "--db", "evergreen_local"); err != nil {
+			if err := smokeRunBinary(exit, "web.service", wd, binary, "service", "web", "--db", "evergreen_local", "--withgraphql"); err != nil {
```

Comment:
The smoke test is meant to be an integration test of the system as we would run in production, so this flag shouldn't be passed by default until we're ready to turn it on. If you'd like to use these as integration tests, add a separate variant similar to what I did for the new security system: https://github.com/evergreen-ci/evergreen/commit/68dfae636b8d1b95102dd71802ffb06cfd7f896c